retail/retail-clickstream-application/main.tf

```diff
@@ -0,0 +1,175 @@
+provider "google" {
```

Comment:
Does the file support comments to add licence header?

Reply:
Added